### PR TITLE
add attachments and embeddables to an email

### LINKED ISSUE #3: adicionar anexo no email
Permitir adicionar anexo nos emails.

### PR DESCRIPTION
**Pull request by mariofts**
Code for add attachments and embeddables to an email.

To embed, just put a key in the template where you want to embed the file.

Fixes #3

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 	<groupId>br.com.caelum.vraptor</groupId>
 	<artifactId>vraptor-simplemail</artifactId>
-	<version>1.2.3</version>
+	<version>1.2.4</version>
```
Comment (luiz) [inline pom.xml]: I think this new feature is enough to a 1.3 version :)

File: src/main/java/br/com/caelum/vraptor/simplemail/template/DefaultTemplateMail.java [hunk 2]
```diff
@@ -59,5 +75,37 @@ protected HtmlEmail prepareEmail(String name, String toMail) {
...
+	public TemplateMail embed(String name, URL url) {
+		toEmbed.put(name, new URLDataSource(url));
+		return this;
+	}
+	
+	@Override
+	public TemplateMail attach(String name, File file) {
+		toAttach.put(name, new FileDataSource(file));
+		return this;
+	}
```
Comment (luiz) [inline src/main/java/br/com/caelum/vraptor/simplemail/template/DefaultTemplateMail.java:109]: Just for completeness: add an attach(String,URL) method too ;)

Comment (mariofts) [inline src/main/java/br/com/caelum/vraptor/simplemail/template/DefaultTemplateMail.java:109]: done here: 6832bdf

**Comment by luiz** [comment]
Merged. Thanks! :)

**Pull request merged**